cmake/README.md

```diff
@@ -26,17 +26,25 @@ TBB_BUILD_APPLE_FRAMEWORKS - Enable the Apple* frameworks instead of dylibs, onl
 
 ### Preparation
 
-To perform an out-of-source build, create a build directory and go there:
+To perform an out-of-source build, create a build directory outside of the oneTBB source directory and go there.
 
 ```bash
 mkdir /tmp/my-build
 cd /tmp/my-build
 ```
 
+Or you may also create a build directory within the oneTBB source directory.
```

Comment:
Purpose of this addition is not clear. A temporary build directory can be created in an arbitrary location - this is what is stated in the original message above. 

Probably it is worth to add a better example instead of changing the general instructions.

Reply:
Yea. What do you think about changing [the examples here](https://github.com/oneapi-src/oneTBB/blob/master/INSTALL.md#example-of-installation) to be out of source build?